spec.emu

```diff
@@ -71,7 +115,7 @@ contributors:
       1. Perform ? RequireInternalSlot(_M_, [[WeakMapData]]).
       1. Let _entries_ be the List that is _M_.[[WeakMapData]].
       1. <del>If Type(_key_) is not Object, return *false*.</del>
-      1. <ins>If IsObjectOrSymbol(_key_) is *false*, return *false*.</ins>
+      1. <ins>If ! CanBeHeldWeakly(_key_) is *false*, return *false*.</ins>
```

Comment:
```suggestion
      1. <ins>If CanBeHeldWeakly(_key_) is *false*, return *false*.</ins>
```
same throughout, since this AO doesn’t return a completion record

Reply:
thanks!